wallet/src/main/scala/org/bitcoins/wallet/internal/TransactionProcessing.scala

```diff
@@ -400,6 +410,20 @@ private[wallet] trait TransactionProcessing extends WalletLogger {
 
           for {
             _ <- insertIncomingTransaction(transaction, totalIncoming)
+            prevTagDbs <- addressTagDAO.findTx(transaction.txIdBE,
+                                               networkParameters)
+            prevTags = prevTagDbs.map(tagDb =>
+              AddressTag(tagDb.tag.tag, Some(tagDb.tagType)))
```

Comment:
can we rename `AddressTag.tag` -> `AddressTag.name` ? So we don't have to do `tagDb.tag.tag` which is confusing